netket/vqs/mc_state.py

```diff
@@ -727,10 +724,10 @@ def grad_expect_operator_kernel(
         σ = σ.reshape((-1, σ_shape[-1]))
 
     has_aux = mutable is not False
-    if not has_aux:
-        out_axes = (0, 0)
-    else:
-        out_axes = (0, 0, 0)
+    # if not has_aux:
+    #    out_axes = (0, 0)
+    # else:
+    #    out_axes = (0, 0, 0)
```

Comment:
What is the purpose of these changes?

Reply:
that is unused code.
Flake8 complains it's unused.
I want to keep it to remind myself to get back to it (it's needed for models that have `has_aux=True`).